graphics/materialx/Portfile

```
@@ -79,6 +79,12 @@ post-destroot {
     if {[file exists ${destroot}${prefix}/README.md]} {
         move ${destroot}${prefix}/README.md $mtlx_share_path
     }
+
+    fs-traverse f $mtlx_share_path/resources {
+        if {[file isfile $f] && [regexp {CMakeLists.txt} $f]} {
```

Seems like just a string equality comparison of the filename would have been simpler and more accurate?
```suggestion
        if {[file isfile $f] && [file tail $f] eq "CMakeLists.txt"} {
```

---

Yes, it would be simpler. I didn't know you could do that.

---

Yeah `file tail` gets used a lot for this kind of thing. Tcl has lots of other `file` commands that you may find useful from time to time. See https://www.tcl.tk/man/tcl8.5/TclCmd/file.htm